cli/crypto.py

```diff
@@ -1,30 +1,30 @@
 from dataclasses import dataclass
 from random import randint
-from typing import Tuple, List, Dict, Any
+from typing import Any, Dict, List, Tuple, cast
 
 from Crypto.Cipher import AES, PKCS1_OAEP
+from Crypto.Cipher._mode_eax import EaxMode
 from Crypto.PublicKey.RSA import RsaKey
 from Crypto.Random import get_random_bytes
 from eth2deposit.key_handling.keystore import Keystore, Pbkdf2Keystore
-from eth_typing import BLSPubkey, BLSSignature
-from py_ecc import optimized_bls12_381
-from py_ecc.bls import G2ProofOfPossession as bls_pop
+from eth_typing.bls import BLSPubkey, BLSSignature
```

Comment:
eth_typing has the top level import too but no `__all__` with public interface. mypy doesn't like such approach. Thus I changed the import to point to the place where the thing that's imported is defined.